clean.sh

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env bash
+
+usage()
+{
+    echo "Usage: clean [options]"
+    echo "Cleans the local dev environment."
+    echo
+    echo "  -b         Delete bin directory"
+    echo "  -p         Delete packages directory"
+    echo "  -t         Delete Tools directory"
+    echo "  -w         Remove all untracked files in the working tree, including the bin, packages, and Tools directories"
```

Comment:
I did consider what you have here when I did the clean.cmd and after thinking about the flow a little bit I think you should probably mimic what I did and create a similar sync.sh script as well.